recipes/impedancefitter/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "impedancefitter" %}
+{% set version = "2.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/impedancefitter-{{ version }}.tar.gz
+  sha256: 30eac8eb8d7230cc948eefdfad638239f09af0efb40d96a6e0e8c758811e6279
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python ==3.8
```

Comment:
Upstream only lists >=3.7. Why are you restricting this?

https://github.com/j-zimmermann/ImpedanceFitter/blob/4a731940336718946e49a9cd7bab035d2a8edc01/setup.py#L29

Reply:
Sorry, this was a not-so-clever attempt to build the package with Python 3.8.
The automatic checks fail because Python 3.10 cannot install schemdraw.
I haven't experienced this in my github workflow (https://github.com/j-zimmermann/ImpedanceFitter/tree/v2.0.4) nor on my local machines.

Could you please help me with that?

Reply:
The versions currently available on conda-forge are only 0.14 and 0.15. Since you're also upstream, is it possible to use that?

https://anaconda.org/conda-forge/schemdraw/files

Reply:
I updated this version. Now it says that some tests fail.
Tbh I have no idea why that is and I apologise for taking so much of your time.
I run tests for every commit on my local machines and on github. They do not fail there. I can only see some warning about an emcee version not properly installed, which does not make sense to me because this version is available through conda-forge.

Reply:
Can you please advise me what the best way is to proceed from here?